### DISCUSSION TITLE
Grafana Bar Possible ?

### DISCUSSION THREAD

**Discussion by Bersaker** [grafana/grafana#37907] [Dashboards, Panels & Visualizations]
Hello, I have a sensor that evaluates 7 states.

0 no charge
2 mistakes
3 normal charge
4 reload
5 trickle charge
7 equalizing charge

I would like to create a bar chart in grafana, which shows me a daily summary of the individual states from midnight to 11.59 p.m.

%%
%%
%% ← 3. Normal charge 2:30
%%
%%
//
// ← 5. trickle charge 1:45
//
$$
$$ ← 4. Reload 0:30
24 hours
Day 1

Is that possible

![Screenshot_20210710-162542](https://user-images.githubusercontent.com/62776321/129442265-9b509c21-c174-4fe1-9521-fd790cd525a4.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
